kinder/pkg/build/alter/alter.go

```diff
@@ -327,6 +327,7 @@ func (c *Context) createAlterContainer(bc *bits.BuildContext) (id string, err er
 		"-v", fmt.Sprintf("%s:%s", bc.HostBasePath(), bc.ContainerBasePath()),
 		// the container should hang forever so we can exec in it
 		"--entrypoint=sleep",
+		"--restart=always", //Always restart the container if it stops
```

Comment:
thank you for the PR.

there should be a space after `//` and before `Always`.

but to ask the more important questions, can you explain why this is needed?
are you seeing cases where we need to restart the alter container?

Reply:
No, I didn't see any errors.  My PR aims to ensure redundancy for container in these case of it is stopped or got an errors.